core/src/main/kotlin/org/eln2/sim/electrical/mna/Node.kt

```diff
@@ -1,18 +1,34 @@
 package org.eln2.sim.electrical.mna
 
 import org.eln2.debug.dprintln
+import org.eln2.sim.electrical.mna.component.*
+import org.eln2.space.Set
 
+/**
+ * A "node", in MNA; the non-resistive connections between [Component]s.
+ *
+ * Aside from identifying [Component]s' connections, the Nodes' potentials (relative to [Circuit.ground]) are computed as a result of the MNA algorithm.
+ */
 open class Node(var circuit: Circuit) : IDetail {
+	/**
+	 * The potential of this node, in Volts, relative to ground (as defined by the [Circuit]); an output of the simulation.
+	 */
 	open var potential: Double = 0.0
+	/**
+	 * The index of this node into the [Circuit]'s matrices and vectors.
+	 */
 	open var index: Int = -1  // Assigned by Circuit
+	/**
+	 * True if this node is ground (defined to have zero potential).
+	 */
 	open val isGround = false
 	var name = "node"
 
 	override fun detail(): String {
 		return "[node val: $potential]"
 	}
 
-	/* Determine which node should prevail when two are merged in a Circuit.
+	/** Determine which node should prevail when two are merged in a Circuit.
 
 	   This is mostly so subclasses of Node (if any) can maintain their existence when merged. The Node returning the
```

Comment:
Hmm. How likely are such subclasses to exist?

Asking because there's a hefty performance penalty to not making the class sealed.

Reply:
`GroundNode` is the big one--it takes preference in all cases. That's the _only_ one right now, though.

Work here mimics what I had to do to get the StationaryShaftNetwork working :)